actions/actions.py

```diff
@@ -65,6 +63,10 @@ class HealthCheckProfileForm(BaseFormAction):
         "age",
         "gender",
         "province",
+        "location",
+        "location_confirm",
+        "latitude",
+        "longitude",
```

Comment:
I don't think that we need to add `latitude` and `longitude` to the list of slots that the form needs to check and ask the user for, especially since they're not in our slot mappings. Just having them in the `domain` should be enough that we could access them, and save data to them